hikari/audit_logs.py

```diff
@@ -327,18 +327,34 @@ class AuditLogEventType(int, enums.Enum):
     INTEGRATION_CREATE = 80
     INTEGRATION_UPDATE = 81
     INTEGRATION_DELETE = 82
+    STAGE_INSTANCE_CREATE = 83
+    STAGE_INSTANCE_UPDATE = 84
+    STAGE_INSTANCE_DELETE = 85
```

Comment:
These are addressed in the stage instances PR - which I have just given Dav a patch to fix up